internal/storage/bundle/conf.go

```diff
@@ -181,17 +87,28 @@ func (conf *Conf) Validate() (outErr error) {
 		outErr = multierr.Append(outErr, err)
 	}
 
-	// SecretKey was renamed to WorkspaceSecret in Cerbos 0.31.0
-	if conf.Credentials.WorkspaceSecret == "" && conf.Credentials.SecretKey != "" {
-		conf.Credentials.WorkspaceSecret = conf.Credentials.SecretKey
+	if err := conf.validateCredentials(); err != nil {
+		outErr = multierr.Append(outErr, err)
 	}
 
-	// InstanceID was renamed to PDPID in Cerbos 0.31.0
-	if conf.Credentials.PDPID == "" && conf.Credentials.InstanceID != "" {
-		conf.Credentials.PDPID = conf.Credentials.InstanceID
+	return outErr
+}
+
+func (conf *Conf) validateCredentials() error {
+	hubConf, err := hub.GetConf()
+	if err != nil {
+		return fmt.Errorf("failed to read Cerbos Hub configuration: %w", err)
 	}
 
-	return outErr
+	switch {
+	case hubConf.Credentials.IsUnset():
+		return conf.Credentials.Validate()
+	case conf.Credentials.IsUnset():
+		conf.Credentials = hubConf.Credentials
+		return conf.Credentials.Validate()
```

Comment:
Should we print a deprecation warning in this case to encourage people to migrate to the `hub` section?